aten/src/ATen/native/native_functions.yaml

```diff
@@ -1099,6 +1099,11 @@
   variants: method
   device_guard: False
 
+- func: empty_quantized(int[] size, Tensor original) -> Tensor
```

Comment:
The name seems a little non-intuitive to me. Maybe make it explicit that original has to be a quantized tensor.

Reply:
sg